app/views/collaborators/_collaborator.html.erb

```diff
@@ -0,0 +1,18 @@
+ 
+ <div class="form-group">
+    <%# this should more properly use a variable set in your controller rather than User.all - for example you might want to limit the list of possible collaborators according to some condition %>
+    <% User.all.each do |user| %>
+      <% #if collaborator = user.collaborator_for(wiki) %>
+      <% if wiki.private? && user != wiki.user && !wiki.collaborators.include?(user) %>
+        <div class="user">
+          <%= user.name %> 
+          <%= link_to  "make collaborator", collaborators_path(wiki_id: wiki.id), class: 'btn btn-primary', method: :post %>
+        </div>
+      <% end %>
+      <% if wiki.private? && user != wiki.user && wiki.collaborators.include?(user) %>
```

Comment:
Lines 6 and 12:

- `wiki.collaborators` returns `Collaborator` objects (not `User` objects)
- `user` is a `User` object
- `wiki.collaborators.include?(user)` will always return false as we're checking if a `User` object is in a collection of `Collaborator` objects. 
- We want to check if `user` is in a collection of `User` objects.
- `wiki.collaborator_users` returns `User` objects. Consider using this instead of `wiki.collaborators`.